configure.ac

```
@@ -1200,6 +1170,32 @@ AS_CASE([$ocaml_cc_vendor,$target],
   [gcc-*,powerpc-*-linux*],
     [oc_ldflags="-mbss-plt"])
 
+# How to build sak
+
+AS_IF([test x"$build" = x"$target" -o x"$target_runnable" = xtrue],
+  [AS_CASE([$target],
+  [*-pc-windows],
+    [SAK_BUILD=m4_normalize(
+      "\$(CC) \$(OC_CFLAGS) \$(CFLAGS) \$(OC_CPPFLAGS) \$(CPPFLAGS) \$(2)
+      /link /out:\$(1) $oc_exe_ldflags \$(OC_LDFLAGS) \$(LDFLAGS)")],
+  [SAK_BUILD=m4_normalize(
+    '$(MKEXE_VIA_CC) $(OC_CFLAGS) $(CFLAGS) $(OC_CPPFLAGS) $(CPPFLAGS)')])
```

```suggestion
    [*-pc-windows],
      [SAK_BUILD=m4_normalize([
        "\$(CC) \$(OC_CFLAGS) \$(CFLAGS) \$(OC_CPPFLAGS) \$(CPPFLAGS) \$(2)
        /link /out:\$(1) $oc_exe_ldflags \$(OC_LDFLAGS) \$(LDFLAGS)"])],
    [SAK_BUILD=m4_normalize([
      '$(MKEXE_VIA_CC) $(OC_CFLAGS) $(CFLAGS) $(OC_CPPFLAGS) $(CPPFLAGS)'])])
```
Explicit m4 quotes and fixed indentation.